CMS-Enterprise/easi-app
# [EASI-4656] Duplicate GRB reviewer warning

# EASI-4656

[Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=3905-8011&t=KeZgL30bRdxuu8eX-0)

## Description
- Fixed field validation when trying to add duplicate GRB reviewer
- Added duplicate reviewer warning below GRB member name field
- Unit test for adding duplicate reviewer

## How to test this change
- User should not be able to add duplicate reviewers when using "Add via EUA" tab on GRB Reviewer form
- If attempting to add duplicate reviewer:
  - Warning should render below field
  - Form submit button should be disabled

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
